### PR TITLE
1149 meditrak data time

### PR DESCRIPTION
**Pull request by edmofro**
### Issue #: 
https://github.com/beyondessential/tupaia-backlog/issues/1149, subtasks 10, 11, 13, 14, 18

### CODE REVIEW

File: packages/database/src/migrations/20200218025613-AddDataGroupsToDataSource.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ exports.up = async function(db) {
       JOIN survey ON survey_screen.survey_id = survey.id
       JOIN data_source AS data_group ON survey.code = data_group.code AND data_group.type = 'dataGroup'
       JOIN data_source AS data_element ON question.code = data_element.code AND data_element.type = 'dataElement'
-      WHERE question.type NOT IN ('Instruction', 'SubmissionDate', 'PrimaryEntity')
+      WHERE question.type NOT IN ('Instruction', 'SubmissionDate', 'DateOfData', 'PrimaryEntity')
```
Comment (edmofro) [inline packages/database/src/migrations/20200218025613-AddDataGroupsToDataSource.js:28]: This is just so if anyone uses it for reference, they're aware to also ignore DateOfData

File: packages/meditrak-server/src/routes/importSurveyResponses/importSurveyResponses.js [hunk 2]
```diff
@@ -184,9 +181,12 @@ export async function importSurveyResponses(req, res) {
...
                 });
-              } else if (rowType === ANSWER_TYPES.SUBMISSION_DATE) {
-                // Don't save an answer for submission date rows, instead update the submission date
-                // of the survey response. Note that this will override what is in the Date info row
+              } else if (
+                rowType === ANSWER_TYPES.DATE_OF_DATA ||
+                rowType === ANSWER_TYPES.SUBMISSION_DATE
+              ) {
+                // Don't save an answer for date of data rows, instead update the date_time of the
+                // survey response. Note that this will override what is in the Date info row
```
Comment (edmofro) [inline packages/meditrak-server/src/routes/importSurveyResponses/importSurveyResponses.js:189]: Just whitespace changes - @EMcQ-BES this is where to test you have prettier running

**Review by EMcQ-BES — APPROVED** [review]

**Comment by edmofro** [comment]
Will hold off merging until your base branch is approved @EMcQ-BES

**Comment by edmofro** [comment]
Note to self: I also need to get tests passing before this can be merged

**Comment by edmofro** [comment]
Tests should now be passing

**Comment by EMcQ-BES** [comment]
Merging this so I can use some of the utils you've written

**Pull request merged**